src/kirby/components/alert/alert.component.html

```diff
@@ -0,0 +1,11 @@
+<article  #alertWrapper [ngClass]="iconThemeColor">
+    <h3 class="title">{{ title }}</h3>
+    <div *ngIf="iconName" class="icon-outline">
+        <kirby-icon [name]="iconName" size="lg"></kirby-icon>
+      </div>
+  <div class="message">{{ message }}</div>
+    <div class="buttongroup">
+        <button *ngIf="cancelBtn" attentionLevel="3" kirby-button class="cancel-btn" (click)="onCancel()">{{ cancelBtn }}</button>
+        <button kirby-button class="ok-button" [size]="cancelBtn ? '' : 'lg'"  attentionLevel="1" [isDestructive]="okBtnIsDestructive" (click)="onOk()">{{ okBtn }}</button>
```

Comment:
Should be `null` instead of empty string:
```suggestion
        <button kirby-button class="ok-button" [size]="cancelBtn ? null : 'lg'"  attentionLevel="1" [isDestructive]="okBtnIsDestructive" (click)="onOk()">{{ okBtn }}</button>
```